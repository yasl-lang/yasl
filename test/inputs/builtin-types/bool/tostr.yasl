echo true->tostr()
echo false->tostr()
