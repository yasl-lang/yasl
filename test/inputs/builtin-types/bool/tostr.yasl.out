true
false
[1, 0, 0, 1, 0, 1, 1, 0]
[true, false, false, true, false, true, true, false]
[true, false, false, true, false, true, true, false]
1
0
